csrc/utils.cpp

```diff
@@ -100,6 +102,40 @@ int8_t getCommonDeviceCUDA(
   return found_device ? index : (int8_t)0;
 }
 
+int8_t getCommonDeviceCUDA(
```

Comment:
Check if this can be cleaned up now or if there's still calls into the c10::IValue variant.